rust_src/src/process.rs

```diff
@@ -7,10 +7,17 @@ use lists::{assoc, cdr};
 
 pub type LispProcessRef = ExternalPtr<Lisp_Process>;
 
+impl LispProcessRef {
+    #[inline]
+    fn name(&self) -> LispObject {
+        return LispObject::from_raw(self.name);
+    }
+}
+
 /// Return t if OBJECT is a process.
 #[lisp_fn]
 pub fn processp(object: LispObject) -> LispObject {
-    LispObject::from_bool(object.is_process())
+    return LispObject::from_bool(object.is_process());
```

Comment:
unnecessary (and unidiomatic) change, please revert

Reply:
Added it because of failing build on Travis. I don't like that return either.